app/component/itinerary/navigator/navigator.scss

```diff
@@ -62,6 +69,7 @@
   align-items: center;
   letter-spacing: -0.3px;
   box-shadow: 0 2px 4px 0 rgba(51, 51, 51, 0.2);
+  width: calc(100vw - 16px);
```

Comment:
What's the magic `16px`?

Reply:
The magic is that we for some unkown reason for the moment  need to adjust the width with it's margins. Otherwise margin-right wont work with position: fixed. AFAIK

Reply:
The some unknown reason should be figured out sooner than later

Reply:
With new changes the magic is little less magical